src/schemas/block.json

```
@@ -125,7 +125,8 @@
 	"BlockTag": {
 		"title": "Block tag",
 		"type": "string",
-		"enum": ["earliest", "latest", "pending"]
+		"enum": ["earliest", "latest", "pending", "finalized", "safe", "unsafe"],
+		"description": "Tags \"finalized\" and \"safe\" refer to the corresponding blocks defined by the most recent fork choice state. Tag \"unsafe\" is an alias to the \"latest\"."
```

Is there a compelling reason to use double quotes here instead of single quotes?  The raw JSON is easier to read with single quotes (no need to escape) and should work just as well.
```suggestion
		"description": "Tags 'finalized' and 'safe' refer to the corresponding blocks defined by the most recent fork choice state. Tag 'unsafe' is an alias to the 'latest'."
```